src/main/java/frc/robot/commands/AutoPickup.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import frc.robot.subsystems.BallCamera;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Intake;
+import frc.robot.Constants;
+
+/**
+ * A command that drives the robot to the ball, and picks it up using the intake
+ * 
+ * Requires the drivetrain, intake, and the ball camera
+ */
+public class AutoPickup extends CommandBase {
+  private Drivetrain driveTrain;
+  private Intake intake;
+  private BallCamera camera;
+  private Trajectory trajectory;
+  private ParallelDeadlineGroup commandGroup;
+
+  /** Creates a new AutoPickup. */
+  public AutoPickup(Drivetrain driveTrain, Intake intake, BallCamera camera) {
+    this.driveTrain = driveTrain;
+    this.intake = intake;
+    this.camera = camera;
+
+    this.trajectory = null;
```

Comment:
I don't think you need to initialize these to null here. That should be the case by default.